packages/frontend/core/src/components/affine/enable-affine-cloud-modal/index.tsx

```diff
@@ -50,9 +50,6 @@ export const EnableAffineCloudModal = ({
             ? t['Enable']()
             : t['Sign in and Enable'](),
       }}
-      contentOptions={{
-        ['data-testid' as string]: 'enable-cloud-modal',
-      }}
```

Comment:
I think we should patch the upstream package so that it won't be an issue applying options here

Reply:
Yes, but I don’t think it’s for now, it’s rather urgent.